models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php

```diff
@@ -154,15 +121,15 @@ protected function verifyUriUniqueness(Value $value): void
         }
 
         /** @noinspection PhpUnhandledExceptionInspection */
-        if ((new KernelResource($value->getUri()))->exists() || (new KernelClass($value->getUri()))->exists()) {
+        if ($this->getResource($value->getUri())->exists()) {
```

Comment:
Why we stop checking if the class exists?

```suggestion
        if ($this->getResource($value->getUri())->exists() || $this->getClass($value->getUri())->exists()) {
```